scripts/install_third_party_libs.py

```diff
@@ -175,6 +203,7 @@ def main():
         ('esprima', common.ESPRIMA_VERSION, common.OPPIA_TOOLS_DIR),
         ('PyGithub', common.PYGITHUB_VERSION, common.OPPIA_TOOLS_DIR),
         ('psutil', common.PSUTIL_VERSION, common.OPPIA_TOOLS_DIR),
+        ('protobuf', common.PROTOBUF_VERSION, common.OPPIA_TOOLS_DIR),
```

Comment:
Move this before the previous line, to keep the alphabetical ordering.

Reply:
Done